lint/base_linter/node_linter.py

```diff
@@ -172,9 +172,9 @@ def find_local_executable(self, start_dir, npm_name):
 
                     # Since we've found a valid 'package.json' as our 'project_root'
                     # exhaust outer loop looking just for installations.
-                    for path in paths:
+                    for path_ in paths:
```

Comment:
I don't like the look of a dangling underscore, but 🤷‍♂ Just `p` because it's short lived?

Reply:
Yeah, okay

Reply:
Single char var is just as ugly. Not important enough.

Reply:
Excellent 👌🏻